src/utils/validation_utils.py

```diff
@@ -13,13 +16,16 @@
 
 def validate_dataset(
     da,
+    filename,
     lat_var="y",
     lon_var="x",
     lat_range=(90, -90),
     lon_range=(-180, 180),
     num_attrs=15,
 ) -> bool:
     """Validate the dataset meets expected criteria."""
+    date, date_type = get_datetime_from_filename(filename, return_type=True)
+    date_type = "valid" if date_type == "v" else "issued"
```

Comment:
Ah just tested this for `imerg` and realized it actually fails since it gets the `_` date_type, which then infers it as being `issued`, which is incorrect. 

We should flip this and only assume `issued` if it's explicit, then `valid` otherwise: 

`date_type = "issued" if date_type == "i" else "valid"`